### ISSUE TITLE
Duplicated error messages when using `meta={'unique_together': ...}`

### ISSUE THREAD

**Issue by yakky** [django-parler/django-parler#236]
When using unique together on translated fields, error message is duplicated, apparently because both form  (https://github.com/django-parler/django-parler/blob/master/parler/forms.py#L169) and model (https://github.com/django-parler/django-parler/blob/master/parler/models.py#L654) run the same `translation.validate_unique()` check

This might also be a flaw in my model implementation,thus  I need to investigate better

See nephila/djangocms-blog#466 for context

**Comment by vdboor** [comment]
Does PR #244 fix this?

**Comment by mireq** [comment]
No, but it shouldn't be difficult to deduplicate with own mixin (not tested):

```python
class DeduplicatedErrorsFormMixin(object):
    def _update_errors(self, errors):
        if not hasattr(errors, 'error_dict'):
            return super()._update_errors(errors)

        deduplicated_error_messages = {}

        for field, messages in errors.error_dict.items():
            for message in messages:
                if self.has_error(field, message.code):
                    continue
                deduplicated_error_messages.setdefault(field, [])
                deduplicated_error_messages[field].append(message)

        return super()._update_errors(ValidationError(deduplicated_error_messages))
```

I am using a little more complicated code that changes error messages from "field is not unique for this language and site" to "field is not unique" because language and site_id are not editable in my form. To accomplish this I needed a complete ValidationError object object with unique constraints.
